cpp/src/aztec/honk/sumcheck/relations/relation.test.cpp

```diff
@@ -19,11 +18,12 @@ template <class FF> class SumcheckRelation : public testing::Test {
   public:
     template <size_t t> using Univariate = Univariate<FF, t>;
     template <size_t t> using UnivariateView = UnivariateView<FF, t>;
+    static constexpr size_t FULL_RELATION_LENGH = 5;
```

Comment:
Definitely understand if you don't want to do this in this PR but just some thoughts: This `FULL_RELATION_LENGH` is necessarily tied to the actual full relation length so the hard coding is kind of brittle and has tripped me up on a few occasions. It would be nice to compute this programmatically instead then update`compute_mock_extended_edges` to create appropriately sized mock univariates. (We could do this by either directly generating the full sized univariate or just defining length 2 univariates then extending them appropriately).

Reply:
Agreed that this would be nice. I think that we can get this done as part of Arijit's PR since he introduced a lot more functionality to the way the test is performed. Left a more detailed TODO for now